examples/android/README.md

```diff
@@ -26,5 +26,5 @@ $ # follow the instructions in 'How to Build'
 **3 Install the app**
```

Comment:
Since you have switched to the snapshot version of grpc, shall we make it clear that the user needs to do `./gradlew install` in the previous step?